actioncable/lib/action_cable/channel/base.rb

```diff
@@ -177,6 +177,15 @@ def unsubscribe_from_channel # :nodoc:
         end
       end
 
+      def transmit_client_error(error_type:) # :nodoc:
+        logger.error "Subscription on #{self.class.name} already exists for: #{@identifier}"
+        logger.info "#{self.class.name} is transmitting a client error"
+
+        ActiveSupport::Notifications.instrument("transmit_client_error.action_cable", channel_class: self.class.name) do
+          data = { identifier: @identifier, type: ActionCable::INTERNAL[:message_types][:client_error] }
+          connection.transmit data.merge!(message: { error_type: error_type })
```

Comment:
Shouldn't we document the error transmits somewhere? And what kind of errors can be transmitted?